tests/integration/publish_tests.rs

```diff
@@ -215,6 +215,43 @@ itest!(config_flag {
   http_server: true,
 });
 
+#[test]
+fn ignores_gitignore() {
```

Comment:
How does someone include a file that's gitignored? Can we write a test for that before landing?

Reply:
Added a test for `publish.includes`. It takes precedence over .gitignore.